controller/engine_controller.go

```diff
@@ -48,6 +49,9 @@ var (
 	EngineMonitorConflictRetryCount = 5
 
 	purgeWaitIntervalInSecond = 24 * 60 * 60
+
+	// restoreMaxInterval: deleting the backup of big size volume takes a long time for retain policy and restoring backups would be in backoff period.
+	restoreMaxInterval = 1 * time.Hour
```

Comment:
nit: use constant instead of var here. (it seems all variables here should be in const instead)

Reply:
also, I think this kind of implicit timeout should be documented to let users better know what happens under the hood. (the same as expansion backup)